src/buildercore/fastly/vcl/strip-non-journal-cookies.vcl

```diff
@@ -0,0 +1,7 @@
+declare local var.journal_cookie STRING;
+set var.journal_cookie = req.http.Cookie:journal;
```

Comment:
The cookie name is a Pillar value. But we don't currently vary it.

Reply:
Was introduced before the possibility of VCL templates was implemented, so the cookie name `journal` could be passed in now - but the behavior is very specific to Journal anyway

Reply:
For now it's ok; there's a pretty standard VCL snippet for whitelisting one or more cookies (with exact or partial names), which we can always switch to if something else needs it.